### PR TITLE
[JF] add diamons to the game

### PR DESCRIPTION
**Pull request by tatosweb**
@Shinkei please help me to review this :dagger:

### CODE REVIEW

File: js/game.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
 var game = new Phaser.Game(800, 600, Phaser.AUTO, '', { preload: preload, create: create, update: update });
-var platform, player, cursor, stars, score, baddie, baddie_go_right, sfxAudio, scoreText, sfxDeath;
+var platform, player, cursor, stars, score, baddie, baddie_go_right, sfxStar, scoreText, sfxDeath, diamonds;
```
Comment (Shinkei) [inline js/game.js]: creo que falto la variable sfxDiamond

File: js/game.js [hunk 2]
```diff
@@ -19,10 +21,12 @@ function create() {
   game.add.sprite(0, 0, 'sky');
 
   //  Audio setup
-  sfxAudio = game.add.audio('sfx:ping');
-  sfxAudio.allowMultiple = true;
+  sfxStar = game.add.audio('sfx:ping');
+  sfxStar.allowMultiple = true;
   sfxDeath = game.add.audio('sfx:death');
   sfxDeath.allowMultiple = true;
+  sfxDiamod = game.add.audio('sfx:diamond');
```
Comment (Shinkei) [inline js/game.js]: typo en sfxDiamod debería ser sfxDiamond

File: js/game.js [hunk 3]
```diff
@@ -70,7 +74,20 @@ function create() {
...
+  diamonds = game.add.group();
+  diamonds.enableBody = true;
+
+  // create the diamonds
+  for (var i = 0; i < 6; i++) {
+    var diamond = diamonds.create(i * 120, 20, 'diamond');
+    diamond.body.gravity.y = 200;
+    diamond.body.bounce.y = 0.2 + Math.random() * 0.4;
+    // add sound to each diamond object
+    sfxDiamod.addMarker('diamond',0,1.5);
```
Comment (Shinkei) [inline js/game.js]: typo en sfxDiamod

**Comment by tatosweb** [comment]
@Shinkei done, please review :1st_place_medal:

**Pull request merged**